File: Wordplay/python/2.py
# What are all of the words containing an X and a Y and a Z?
def main():
    in_file = open("../sowpods.txt", "r")
    for words in in_file:
        if("X" in words and "Y" in words and "Z" in words):
            print(words, end="")

if __name__ == '__main__':
    main()

File: Wordplay/js/2.js
// What are all of the words containing an X and a Y and a Z?

let main = () => {

    const fs = require("fs");

    let buffer_arry = fs.readFileSync("Wordplay/sowpods.txt", "utf-8");
    let in_file = buffer_arry.split("\n");

    for(let i = 0; i < in_file.length; i++) {
        if(in_file[i].includes("X") && in_file[i].includes("Y")
        && in_file[i].includes("Z")) {
            console.log(in_file[i])
        };
    };
};

main();
